### PR TITLE
Update the minimum tls version for python client to avoid https shake verify issue.

### PR DESCRIPTION
**Pull request by magiWei**
@lei-flex @tylerflex  There is a pr that users can set https ssl version, the ticket from @lei-flex  https://flow360.atlassian.net/browse/SCEM-5072

### CODE REVIEW

**Review by lei-flex — CHANGES_REQUESTED** [review]

**Comment by lei-flex** [comment]
@tylerflex @tomflexcompute could you also follow up to update the docs for this change, in Q&A part?

File: tidy3d/web/core/http_util.py [hunk 1]
```diff
@@ -142,33 +144,53 @@ def wrapper(*args, **kwargs):
...
+    def reinit(self):
+        """Reinitialize the session."""
+        ssl_version = Env.current.ssl_version
+        if ssl_version:
+            self.session.mount("https://", TLSAdapter())
+
     @http_interceptor
     def get(self, path: str, json=None):
         """Get the resource."""
+        self.reinit()
```
[the remark this answers is not in the document]
Comment (magiWei) [reply tidy3d/web/core/http_util.py:176]: Good idea, a variable named `REINITIALIZED` is added to identify whether it has been reinited.

**Review by lei-flex — APPROVED** [review]
ship it.

**Comment by tylerflex** [comment]
hi @magiWei after I merged the other PR, there are some merge conflicts here, could you fix them up and then I will take a last look and merge this, thanks!

**Pull request merged**